Carmunication/Extensions/SessionStartView+Extension.swift

```diff
@@ -0,0 +1,285 @@
+
+        // 변환된 날짜를 UILabel에 표시
+        let dateLabel: UILabel = {
+            let lbl = UILabel()
+            lbl.text = formattedDate
+            lbl.textColor = ColorTheme().blue8
+            lbl.font = CarmuFont().body2Long
+            return lbl
+        }()
+
+        journeySummaryView.addSubview(dateLabel)
+
+        dateLabel.snp.makeConstraints { make in
+            make.leading.top.equalTo(journeySummaryView).inset(20)
+        }
+
+        // 여정 요약 (중앙 부분)
+        if groupData == nil {
+            let comment: UILabel = {
+                let lbl = UILabel()
+                lbl.text = "초대하거나 받은 여정이 없어요...\n친구와 함께 여정을 시작해 보세요!"
+                lbl.font = CarmuFont().headline1
+                lbl.textAlignment = .center
+                lbl.textColor = SemanticColor().textBody
+                lbl.numberOfLines = 0
+                return lbl
+            }()
+
+            journeySummaryView.addSubview(comment)
+            comment.snp.makeConstraints { make in
+                make.centerX.equalTo(journeySummaryView)
+                make.top.equalTo(journeySummaryView).inset(80)
+            }
+        } else {
+            setRouteComponent()
+        }
+    }
+
+    func setDayAndPerson() {
+        setDayViewComponent()
+        setPersonCountViewComponent()
+
+        journeySummaryView.addSubview(dayView)
+        dayView.snp.makeConstraints { make in
+            make.leading.equalTo(journeySummaryView).inset(12)
+            make.bottom.equalTo(journeySummaryView).inset(70)
+            make.height.equalTo(40) // 높이를 설정할 수 있습니다.
+        }
+
+        journeySummaryView.addSubview(personCountView)
+        personCountView.snp.makeConstraints { make in
+            make.trailing.equalTo(journeySummaryView).inset(12)
+            make.bottom.equalTo(journeySummaryView).inset(70)
+            make.height.equalTo(40)
+            make.width.equalTo(dayView.snp.width)
+        }
+        // 크기를 같게 설정한 뒤, 두 뷰 사이의 padding을 10 추가
+        dayView.snp.makeConstraints { make in
+            make.trailing.equalTo(personCountView.snp.leading).offset(-10)
+        }
+    }
+
+    func setDayViewComponent() {
+        let calendarImage: UIImageView = {
+            let imageView = UIImageView()
+            if let image = UIImage(systemName: "calendar") {
+                imageView.image = image
+                imageView.tintColor = SemanticColor().accPrimary
+            }
+            return imageView
+        }()
+        let dayLabel: UILabel = {
+            let lbl = UILabel()
+            lbl.textAlignment = .center
+            lbl.textColor = SemanticColor().textPrimary
+            lbl.font = CarmuFont().body2Long
+            lbl.text = groupData == nil ? "---" : "주중 (월 ~ 금)"
+            return lbl
+        }()
+
+        dayView.addSubview(calendarImage)
+        calendarImage.snp.makeConstraints { make in
+            make.leading.equalTo(dayView).offset(12)
+            make.centerY.equalTo(dayView)
```

Comment:
superview를 기준으로 오토레이아웃을 잡는다면 아래와 같이 쓰는게 가독성이 좋을 것 같아요 😁

```suggestion
            make.leading.equalToSuperview().offset(12)
            make.centerY.equalToSuperview()
```